src/containers/Login/Login.js

```diff
@@ -1,9 +1,9 @@
 import React, { Component } from 'react';
-import { getUser } from '../../helpers/getUser';
+import { getUser } from '../../helpers/requests';
```

Comment:
We can combine {getUser} and {getUserFavorites} because they are from the same file